config/application.rb

```diff
@@ -152,6 +152,14 @@ class Application < Rails::Application
       #{config.root}/app/jobs
     )
 
+    initializer "ofn.reports" do |_app|
+      module ::Reporting; end
+      loader = Zeitwerk::Loader.new
+      loader.push_dir("#{Rails.root}/lib/reporting", namespace: ::Reporting)
```

Comment:
Would these files not get loaded otherwise?

Reply:
They wouldn't be loaded at boot time, and you would have to use `require "reporting/whatever"` at the top of every file where you wanted to use one of these classes. So we can skip that and just tell Zeitwerk to load the whole directory into that namespace, then we don't need to call `require` all over the place :+1: